config/file_config.go

```diff
@@ -70,7 +70,7 @@ type HoneycombMetricsConfig struct {
 
 type PeerManagementConfig struct {
 	Type                    string   `validate:"required,oneof= file redis"`
-	Peers                   []string `validate:"required,dive,url"`
+	Peers                   []string `validate:"dive,url"`
```

Comment:
can we do some conditional require here? It is required for `file` type right? Oh wait it has a default! Is that why?

Reply:
It does have a default set 👍